ui/privacy_policy.spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+describe 'Privacy Page', er_137: true do
+  let(:ui) { Ui.new }
+
+  before do
+    ui.privacy_policy.load
+  end
+
+  after do
+    # Capybara.current_session.driver.quit
+    browser = Capybara.current_session.driver.browser
+    browser.manage.delete_all_cookies
+  end
+
+  context 'navigated to by user' do
+    it 'then the Privacy page is displayed' do
+       expect(ui.privacy_policy).to have_xpath("//h1[contains(text(),'Privacy policy')]")
```

Comment:
I haven't tried this but I'd expect we could do something like `ui.privacy_policy.header` and `have_text`, is that possible?

Reply:
its would require the **text** be identified as the element - the asserting against the element, but when I attempted - It wasn't working will refactor - when I have investigated solution